File: common/src/main/java/com/wdz/common/net/response/CollectWebResponse.java
package com.wdz.common.net.response;

public class CollectWebResponse {
    public Data data;

    @Override
    public String toString() {
        return "CollectWebResponse{" +
                "data=" + data +
                '}';
    }

    class Data{
        public String desc;

        public String icon;

        public int id;

        public String link;

        public String name;

        public int order;

        public int userId;

        public int visible;

        @Override
        public String toString() {
            return "Data{" +
                    "desc='" + desc + '\'' +
                    ", icon='" + icon + '\'' +
                    ", id=" + id +
                    ", link='" + link + '\'' +
                    ", name='" + name + '\'' +
                    ", order=" + order +
                    ", userId=" + userId +
                    ", visible=" + visible +
                    '}';
        }
    }
}

File: kt_common/src/main/java/com/wdz/ktcommon/http/response/CollectWebResponse.kt
package com.wdz.ktcommon.http.response

data class CollectWebResponse(var data: Data) {


    data class Data(
        var desc: String,
        var icon: String,
        var id: Int,
        var link: String,
        var name: String,
        var order: Int,
        var userId: Int,
        var visible: Int
    ) {

    }
}